src/utils.py

```diff
@@ -128,3 +130,36 @@ def persist_repo_in_db(data):
 def scan_repo(data):
     """Scan function."""
     return True
+
+def server_run_flow(flow_name, flow_args):
+    """Run a flow.
+    :param flow_name: name of flow to be run as stated in YAML config file
+    :param flow_args: arguments for the flow
+    :return: dispatcher ID handling flow
+    """
+    current_app.logger.debug('Running flow {}'.format(flow_name))
+    start = datetime.datetime.now()
+
+    init_celery(result_backend=False)
+    dispacher_id = run_flow(flow_name, flow_args)
+
+    elapsed_seconds = (datetime.datetime.now() - start).total_seconds()
+    current_app.logger.debug("It took {t} seconds to start {f} flow.".format(
+        t=elapsed_seconds, f=flow_name))
+    return dispacher_id
+
+
+#To integrate with Aagams workflow
+def server_create_component_bookkeeping(ecosystem, name, version, user_profile):
```

Comment:
This needs to be changed @GeetikaBatra

Reply:
@samuzzal-choudhury This is for a reference. Will remove it from this PR as I will be opening a new PR for worker flow.